### PR TITLE
Integrate Ace as the Wang code editor

### PR DESCRIPTION
**Pull request by darius98**

### CODE REVIEW

**Comment by darius98** [comment]
Resolves Issue #1.

File: panda/lib/components/text_editor/text_editor.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<material-spinner *ngIf="!loaded"></material-spinner>
+<div *ngIf="loaded" [id]="uniqueId" style="height: 320px; width: 100%;">
```
Comment (bciobanu) [inline panda/lib/components/text_editor/text_editor.html]: It would be best if we moved the styles in a separate file

**Review by bciobanu — CHANGES_REQUESTED** [review]
Please move all inline style code in a separate files

File: panda/lib/components/text_editor/text_editor.html (hunk 1 above)
Comment (darius98) [reply panda/lib/components/text_editor/text_editor.html]: Resolved in commit bcbe59396551c5fb9976fe12ade5168c04ae66e2.

**Review by bciobanu — APPROVED** [review]

**Review by AndreiNet — APPROVED** [review]

**Pull request merged**